### PR TITLE
June 16 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 0799474 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-314.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20230616_amplify.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+| Apigee Edge                                | 0.1.14                  |
+| Mulesoft Anypoint platform v3              | 1.1.21                  |
+
+#### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+##### Enhancements for Axway Agent SDK
+
+* **Security patches** - An update has been made to the security patche of some dependent libraries.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230616_amplify.md:32]: patches

File: content/en/docs/amplify_relnotes/20230616_amplify.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+The Axway API Management Gateway agents have been updated with the Axway Agent SDK. There are no other new enhancements or bug fixes in this release.
+
+#### Embedded agent
+
+The Embedded Agent has been updated with the Axway Agent SDK. There are no other new enhancements in this release.
+
+##### Bug fixes for Embedded Agent
+
+* **Provisioning of Credentials** - An fix was made to the handling of credential events (e.g. update and deletin reqeuts).
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230616_amplify.md:52]: typo: deletin reqeuts -> deleting requests

File: content/en/docs/amplify_relnotes/20230616_amplify.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+There are no new features, bug fixes, or enhancements in this release.
+
+### Topology
+
+There are no new features, or bug fixes in this release.
+
+#### Feature updates for Topology
+
+* **Environment Details** - If Compliance Validation is enabled for an Environment, the Design and/or Security grading can be displayed in the table of Services as columns.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230616_amplify.md:88]: When Compliance Validation is enabled for an Environment, the Design and / or Security gradin are displayed in the Services table.

File: content/en/docs/amplify_relnotes/20230616_amplify.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+### Asset Catalog
+
+The following enhancements and bug fixes are available in this release.
+
+* **Add/Edit Asset** - If Compliance Validation is enabled, the Design and/or Security grading can be displayed in the table as columns at the Group Resources step.  This enables the provider to make an more informed decision whether to add a resource based on its compliance grading.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230616_amplify.md:94]: If Compliance Validation is enabled, the Design and/or Security grading are displayed during creation / editing of an asset in the Group Resource screen. This enables the provider to make an informed decision whether to add a resource to an asset based on its compliance score.

**Review by georgiana-caras — COMMENTED** [review]
Partial review for the asset and service registry sections. I need to add content to the Product Foundry.

**Pull request merged**